docs/lua-garbage-collector.md

```
@@ -0,0 +1,165 @@
+luaGcStepSize = 200              -- step size (KB to process per step)
+luaGcFullCollectOnStartup = true -- full GC after loading all scripts
+luaGcFullCollectOnShutdown = true-- full GC before closing Lua state
+luaGcWarnMemoryKb = 262144       -- warn threshold (256 MB)
+luaGcCriticalMemoryKb = 524288   -- critical threshold (512 MB)
+```
+
+## Admin Commands
+
+Use `/luagc` as GOD (account type 6):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)